apps/frontend/src/news/__tests__/NewsList.test.tsx

```diff
@@ -1,46 +1,52 @@
-import nock from 'nock';
+import { RecoilRoot } from 'recoil';
+import { Suspense } from 'react';
 import { render, waitFor } from '@testing-library/react';
 import { MemoryRouter, Route } from 'react-router-dom';
-import { authTestUtils } from '@asap-hub/react-components';
+
 import { renderHook } from '@testing-library/react-hooks';
 import { createNewsResponse } from '@asap-hub/fixtures';
 import { usePagination, usePaginationParams } from '../../hooks';
 
 import NewsAndEventsPage from '../Routes';
-import { API_BASE_URL } from '../../config';
-
-const newsRequest = ({
-  pageSize,
-  numberOfItems,
-}: {
-  pageSize: number;
-  numberOfItems: number;
-}) =>
-  nock(API_BASE_URL, {
-    reqheaders: { authorization: 'Bearer token' },
-  })
-    .get('/news')
-    .query(true)
-    .reply(200, {
-      total: numberOfItems,
-      items: Array.from({ length: pageSize }).map((_, idx) =>
-        createNewsResponse(idx + 1),
-      ),
-    });
-
-const renderPage = async () => {
+import { refreshNewsItemState } from '../state';
+import { Auth0Provider, WhenReady } from '../../auth/test-utils';
+
+import { getNews } from '../api';
+
+jest.mock('../api');
+
+const mockGetNews = getNews as jest.MockedFunction<typeof getNews>;
+
+const getMockedResponse = (pageSize = 10, numberOfItems = 10) => ({
+  total: numberOfItems,
+  items: Array.from({ length: pageSize }, (_, idx) =>
+    createNewsResponse(idx + 1),
+  ),
+});
+beforeEach(() => {
+  mockGetNews.mockClear();
+  mockGetNews.mockResolvedValue(getMockedResponse(10, 10));
```

Comment:
Why do you mock `getNews` response twice?

https://github.com/yldio/asap-hub/blob/6c2f29bcc29b8b08da402a2be4c5f09fd2a15b4b/apps/frontend/src/news/__tests__/NewsList.test.tsx#L70

https://github.com/yldio/asap-hub/blob/6c2f29bcc29b8b08da402a2be4c5f09fd2a15b4b/apps/frontend/src/news/__tests__/NewsList.test.tsx#L79

https://github.com/yldio/asap-hub/blob/6c2f29bcc29b8b08da402a2be4c5f09fd2a15b4b/apps/frontend/src/news/__tests__/NewsList.test.tsx#L101